REL/generate_train_test.py

```diff
@@ -334,9 +331,7 @@ def __save(self, mentions_dataset, file_name):
         """
 
         with open(
-            "{}/{}/generated/test_train_data/{}.pkl".format(
-                self.base_url, self.wiki_version, file_name
-            ),
+            os.path.join(self.base_url, self.wiki_version, '/generated/test_train_data/', file_name, 'pkl'),
```

Comment:
Same here, I don't think this results in "blabla.pkl". I think it should be `f"{file_name}.pkl"`